_config.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+// Ensure compatibility with PHP 7.2 ("object" is a reserved word),
+// with SilverStripe 3.6 (using Object) and SilverStripe 3.7 (using SS_Object)
+if (!class_exists('SS_Object')) {
+  class_alias('Object', 'SS_Object');
```

Comment:
Sorry, four spaces (nit picking)

Reply:
No problem, I was used to tabs in SS3.

Reply:
We bless them and thank them duly for their long time support and hard work